File: YouTubePlayer/src/main/java/com/pierfrancescosoffritti/youtubeplayer/ui/menu/YouTubePlayerMenu.java
package com.pierfrancescosoffritti.youtubeplayer.ui.menu;

import android.view.View;

public interface YouTubePlayerMenu {
    void showMenu(View anchorView);
    void addItem(MenuItem menuItem);
    void removeItem(int itemIndex);
    int getItemCount();
}

File: core-custom-ui/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/customui/menu/YouTubePlayerMenu.kt
package com.pierfrancescosoffritti.androidyoutubeplayer.core.customui.menu

import android.view.View

interface YouTubePlayerMenu {
  val itemCount: Int
  fun show(anchorView: View)
  fun dismiss()

  fun addItem(menuItem: MenuItem): YouTubePlayerMenu
  fun removeItem(itemIndex: Int): YouTubePlayerMenu
  fun removeItem(menuItem: MenuItem): YouTubePlayerMenu
}
